### PR TITLE
Adds PAT check to shared pr check

### PR DESCRIPTION
**Pull request by supertopher**
## What
adds PAT check to issue checker

### CODE REVIEW

File: .github/workflows/shared-pulls.yml [hunk 1]
```diff
@@ -4,13 +4,34 @@ on:
...
+        id: variables
+        run: |
+          ./tools/bin/find_non_rate_limited_PAT \
+            ${{ secrets.AIRBYTEIO_PAT }} \
+            ${{ secrets.OSS_BUILD_RUNNER_GITHUB_PAT }} \
+            ${{ secrets.SUPERTOPHER_PAT }} \
+            ${{ secrets.DAVINCHIA_PAT }}
+
   shared-pulls:
+    name: "Run github actions to organize issues.  SAFE_TO_PUSH_WHEN_FAILING"
```
Comment (supertopher) [inline .github/workflows/shared-pulls.yml:26]: @timroes would this label help you in the future?

@bleonard is this label true and do you have thoughts about it?

File: .github/workflows/shared-pulls.yml [hunk 2]
```diff
@@ -4,13 +4,34 @@ on:
...
+        id: variables
+        run: |
+          ./tools/bin/find_non_rate_limited_PAT \
+            ${{ secrets.AIRBYTEIO_PAT }} \
+            ${{ secrets.OSS_BUILD_RUNNER_GITHUB_PAT }} \
+            ${{ secrets.SUPERTOPHER_PAT }} \
+            ${{ secrets.DAVINCHIA_PAT }}
+
   shared-pulls:
+    name: "Label github issues.  SAFE_TO_PUSH_WHEN_FAILING"
```
Comment (timroes) [inline .github/workflows/shared-pulls.yml:26]: @supertopher Would already be helpful for me without the SAFE TO PUSH WHEN FAILING :D Just understanding what it does, (label github issues) would give imho enough context.

**Review by supertopher — COMMENTED** [review]
updated.  Just looking for approval from Brian before I merge.  Thanks for the feedback Tim

File: .github/workflows/shared-pulls.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (bleonard) [reply .github/workflows/shared-pulls.yml]: It's fine. thanks.

**Review by git-phu — APPROVED** [review]
At some point it might be useful to create a shared action for finding PATs

**Comment by supertopher** [comment]
good call Peter.  It's silly to duplicate this as much as we currently are

**Pull request merged**